src/v3/src/components/AuthContainer/AuthContainer.tsx

```diff
@@ -11,13 +11,11 @@
  */
 
 import { Box, useMediaQuery } from '@mui/material';
-import classNames from 'classnames';
 import { FunctionComponent, h } from 'preact';
 
 import style from './style.css';
 
 const AuthContainer: FunctionComponent = ({ children }) => {
-  const classes = classNames('auth-container', 'main-container', style.mainViewContainer);
```

Comment:
I recall this was originally added to help preserve some of the classes that exists in v2 (i.e. if an org is hooking onto the `main-container` class name, to perform custom css changes, this was done to prevent breaking their customizations). Also, I can't recall if the existing v2 tests require one of these classes, if none of these are a concern any longer then I think it is ok to remove, otherwise, we may need to keep it.

Was removing these additional classes needed to make this work?

Reply:
Actually it was not necessary to remove them to make it work, it looked like the widget container was not inheriting any styles from those classes and my thinking was we were going to extract them anyway.  But on second thought I agree I think they still may be needed and I will add them back in.